github/util.py

```diff
@@ -96,6 +92,22 @@ def create_or_update_file(
 
         return response["commit"].sha
 
+    def retrieve_text_file_contents(
+        self,
+        path: str,
```

Comment:
if you opt to rename path to file_path in the previous commit, also rename it here